client/src/components/Header.js

```diff
@@ -124,25 +131,25 @@ export default ({ isAuthenticated, onMenuClick }) => {
                       </li>
                     </>
                   ) : (
-                    <>
-                      <li>
-                        <NavLink activeStyle={activeStyles} to="/auth/login">
-                          Login
+                      <>
+                        <li>
+                          <NavLink activeStyle={activeStyles} to="/auth/login">
+                            Login
                         </NavLink>
-                      </li>
-                      <li className="registerBtn">
-                        <NavLink className="registerLink" to="/auth/signup">
-                          Register
+                        </li>
+                        <li className="registerBtn">
+                          <NavLink className="registerLink" to="/auth/signup">
+                            Register
                         </NavLink>
-                      </li>
-                      <Link to="/feed">
-                        <SvgIcon
-                          src={envelope}
-                          style={{ marginLeft: "1.5rem" }}
-                        />
-                      </Link>
-                    </>
-                  )}
+                        </li>
+                      </>
+                    )}
+                  <li>
+                    <button onClick={() => onFeedbackIconClick()}>
```

Comment:
you can just pass the function as even handler like `onClick={onFeedbackIconClick}`

Reply:
Done :)